### DISCUSSION TITLE
Contribution Guide

### DISCUSSION THREAD

**Discussion by boylerplet** [maximmax42/Discord-CustomRP#94] [General]
I have cloned this project to try and contribute to some feature but i am not experienced in c# development and very new to winforms. I noticed that i had many missing libraries, So without installation of those cannot get started with development. So it would be cool if someone would make a doc on installation procedure for all the modules required and get started with development.

**Comment by maximmax42** [comment]
Hm. If you're using Visual Studio there should be no problems with opening the project up, can't say the same about any other IDE.

**Reply by maximmax42** [reply]
Plus, I never really expected anyone to contribute code.

**Reply by maximmax42** [reply]
Oh and, I keep commits I make through the week locally until I publish new release. Easier to keep track of features I add, but not collaboration friendly.

**Reply by boylerplet** [reply]
All right I got it
